components/dropdown/dropdown-content-mixin.js

```diff
@@ -698,7 +698,9 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		}
 	}
 
-	async __position(ignoreVertical, contentRect) {
```

Comment:
This has been updated to take an `options` argument to keep the API tidy. 

The `ignoreVertical` arg was renamed to `updateAboveBelow` for clarity since we're adding new option to ignore height, which also relates to the vertical dimension.